prow/plugins/lgtm/lgtm.go

```diff
@@ -116,24 +116,14 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Lgtm {
 	fullName := fmt.Sprintf("%s/%s", org, repo)
 	for i := range config.Lgtm {
```

Comment:
Can we replace this loop with the following two loops?

```suggestion
	// First search for repo config
	for _, c := range config.Lgtm {
		if !sets.NewString(c.Repos).Has(fullName) {
			continue
		}
		return &c
	}

	// If you don't find anything, loop again looking for an org config
	for _, c := range config.Lgtm {
		if !sets.NewString(c.Repos).Has(org) {
			continue
		}
		return &c
	}
```

Reply:
I think this looks a lot cleaner. I made the change, thanks so much for reviewing

Reply:
Also, I ran the update-bazel script before committing. Thanks again I appreciate this a lot.